portal-api/server/v1/routes.js

```diff
@@ -49,20 +49,188 @@ const partial2faTokenPassportStrategy = 'login-in-progress';
 // Open router requires no authentication
 const openRouter = express.Router();
 
-// Login route
+/**
+ * @openapi
+ * /login
+ *   post:
+ *     summary: Login
+ *     tags: [Auth]
+ *     description: Login with email and password to receive a JWT token.
+ *    parameters:
+ *      - in: body
+ *        name: username
+ *        schema:
+ *          type: string
+ *      - in: body
+ *        name: password
+ *        schema:
+ *          type: string
+ *    responses:
+ *      200:
+ *        description: Successful login
+ *      401:
+ *        description: Invalid credentials or disabled account
+ *      403:
+ *        description: Account blocked
+ *
+ */
 openRouter.route('/login').post(checkApiKey, users.login);
 
-// 1. Request password reset
+/**
+ * @openapi
+ * /users/reset-password
+ *   post:
+ *     summary: Request password reset
+ *     tags: [Users]
+ *     description: Send a password reset email to the user.
+ *    parameters:
+ *      - in: body
+ *        name: email
+ *        schema:
+ *          type: string
+ *    responses:
+ *      200:
```

Comment:
Please ensure whether the endpoint does return non 2XX response or not when caught please.